user-agreement.md

```diff
@@ -56,16 +56,16 @@ You agree not to do any of the following while using the Site, Services or Frien
 
 ## Termination/Suspension of Accounts
 
-Without limiting other remedies, FriendlyU may at any time suspend, terminate, or refuse to provide you with access to the Site or Services. If FriendlyU terminates, disables or suspends your FriendlyU account for any reason your will not create another FriendlyU account without first receiving FriendlyU's express written permission to do so. In addition, FriendlyU may notify authorities or take any actions it deems appropriate, without notice to you, if FriendlyU suspects or determines, in its own discretion, that you may have or there is a significant risk that you have (i) failed to comply with any provision of these Terms of Service or any policies or rules established by FriendlyU; or (ii) engaged in actions relating to or in the course of using the Site or Services that may be illegal or cause liability, harm, embarrassment, harassment, abuse or disruption for you, FriendlyU users, FriendlyU or any other third parties or the Site or Services.
+Without limiting other remedies, FriendlyU may at any time suspend, terminate, or refuse to provide you with access to the Site or Services. If FriendlyU terminates, disables or suspends your FriendlyU account for any reason your will not create another FriendlyU account without first receiving FriendlyU's express written permission to do so. In addition, FriendlyU may notify authorities or take any actions it deems appropriate, without notice to you, if FriendlyU suspects or determines, in its own discretion, that you may have or there is a significant risk that you have (i) failed to comply with any provision of this User Agreement or any policies or rules established by FriendlyU; or (ii) engaged in actions relating to or in the course of using the Site or Services that may be illegal or cause liability, harm, embarrassment, harassment, abuse or disruption for you, FriendlyU users, FriendlyU or any other third parties or the Site or Services.
```

Comment:
Without limiting other remedies, FriendlyU may at any time suspend, terminate, or refuse to provide you with access to the Site or Services. If FriendlyU terminates, disables or suspends your FriendlyU account for any reason your will not create another FriendlyU account without first receiving FriendlyU's express written permission to do so. In addition, FriendlyU may notify authorities or take any actions it deems appropriate, without notice to you, if FriendlyU suspects or determines, in its own discretion, that you may have or there is a significant risk that you have (i) failed to comply with any provision of this User Agreement or any policies or rules established by FriendlyU; or (ii) engaged in actions relating to or in the course of using the Site or Services that may be illegal or cause liability, harm, embarrassment, harassment, abuse or disruption for you, FriendlyU users, FriendlyU, or any other third parties.